src/pages/community/index.js

```diff
@@ -169,12 +169,14 @@ export default ({ ...data }) => {
                             around the globe.
                         </P>
                         <ProjectActions>
-                            <Button to="#" secondary>
+                            <Button to="/article" primary>
```

Comment:
```suggestion
                            <Button to="/learn" primary>
```

Should this go to the learn page?

Reply:
I think it's going to go to a specific article by Radar